client/components/common/mixins/elementDiscussion.js

```diff
@@ -0,0 +1,55 @@
+
+import { mapActions, mapGetters, mapMutations, mapState } from 'vuex';
+import get from 'lodash/get';
+import { getCommentEvents } from 'tce-core/utils';
+import { mapChannels } from '@/plugins/radio';
+import transform from 'lodash/transform';
+
+const { SAVE, REMOVE, SET_LAST_SEEN } = getCommentEvents();
+
+const COMMENT_EVENTS = [
+  { event: SAVE, action: 'upsertComment' },
+  { event: REMOVE, action: 'removeComment' },
+  { event: SET_LAST_SEEN, action: 'setLastSeenComment' }
+];
+
+export default {
+  computed: {
+    ...mapChannels({ editorBus: 'editor' }),
+    ...mapState('repository/contentElements', { elements: 'items' }),
+    ...mapGetters('repository/comments', ['getUnseenElementComments', 'getComments']),
+    commentsWithinElements() {
```

Comment:
Expose processor for adding them rather than computed.  Use it then in ActivityContent (since there will be other ones as well)